.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,90 @@
+name: Benchmark GTSFM on select datasets
+on: [pull_request]
+jobs:
+  door-dataset-sift:
+    name: Run GTSFM on the select dataset using SIFT front-end
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    env:
+      PYTHON_VERSION: 3.8
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Cache conda env
+        uses: actions/cache@v2
+        env:
+          # Increase this value to reset cache if environment_linux.yml has not changed
+          CACHE_NUMBER: 0
+        with:
+          path: ~/conda_pkgs_dir
+          key:
+            ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{hashFiles('environment_linux.yml') }}
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: gtsfm-v1
+          environment-file: environment_linux.yml
+          python-version: 3.8
+          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
+      - name: Environment setup
+        run: |
+          bash .github/scripts/setup.sh
+          conda info
+      - name: Download Skydio-crane-8 dataset
+        run: |
+          wget --save-cookies cookies.txt 'https://docs.google.com/uc?export=download&id=1mmM1p_NpL7-pnf3iHWeWVKpsm1pcBoD5' -O- \
+            | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1/p' > confirm.txt
```

Comment:
can we add a comment to explain what this does : - ) ?

Reply:
This cookie is for the prompt about the virus scan not being performed. I'll add a comment